pkg/awsv1alpha1/accounts.go

```diff
@@ -29,3 +29,13 @@ func GetAccountCRsToReset(ctx context.Context, cli client.Client) ([]awsv1alpha1
 	}
 	return accountCRs, err
 }
+
+// ResetAccountState sets an account state to Ready
+func ResetAccountState(cli client.Client, account awsv1alpha1.Account) error {
```

Comment:
If we only ever want to set Ready, then this should be named something like `SetAccountStateReady`. The other option I see is leaving the func name, but adding State as an argument